### DISCUSSION TITLE
Windows 10 context menu lines height

### DISCUSSION THREAD

**Discussion by nicolafarina** [valinet/ExplorerPatcher#774] [Ideas]
Hi there,

can be added an option to fix the ridiculous lines height setting of Windows 11 for "classic" context menus? I've noticed that with StartAllBack application, when you disable the new Windows 11 context menu, the old one is fixed already. Can be added a similar look and feel with Explorer Patcher too? Would be really great.

Thanks.

Example:
Windows 11 standard (horrible) extended context menu:
![image](https://user-images.githubusercontent.com/16633188/152179228-61b807df-bcc6-4ee4-9442-0d77734e7e13.png)

Fixed (?) by StartAllBack:
![image](https://user-images.githubusercontent.com/16633188/152179394-29d2c970-5a0f-4884-aa25-06a8946251eb.png)

**Comment by krlvm** [comment]
Microsoft has changed highlight back to light gray and fixed the padding in recent Dev builds, here's the signed msstyles from them:
[aero_dev_channel.zip](https://github.com/valinet/ExplorerPatcher/files/7997147/aero_dev_channel.zip)

**Comment by nicolafarina** [comment]
This is great news!

So we'll have fixed it by design in next release?

Thanks.

**Reply by krlvm** [reply]
You can also replace the aero folder in C:\Windows\Themes right now

**Reply by nicolafarina** [reply]
Any hint how to install it? Probably files are blocked by Windows as the default theme is in use...

**Reply by krlvm** [reply]
I do replace them when OS is running without any issue. However, you can extract them to the desktop, boot from USB, press Shift+F10, type `notepad.exe`, select File -> Open to open a mini-explorer and replace them, but, again, you should be able to do this when OS is running.

**Comment by Anixx** [comment]
How did u get "open in terminal" and "give access to" entries?

**Reply by nicolafarina** [reply]
Nothing special, I suppose simply installing Windows Terminal for its context menu entry. "Give access to" is a Windows default entry.

Take care, this screenshot is taken from an upgraded Windows 10 system to Windows 11, don't know if matters. And it's a Professional edition.

**Comment by OrthodoxWindows** [comment]
You can try to uncheck the box _"Use immersive menus when displaying Windows 10 contect menus"_
